freshmaker/errata.py

```diff
@@ -495,6 +495,28 @@ def get_cve_affected_rpm_nvrs(self, errata_id):
 
         return list(nvrs)
 
+    def get_cve_affected_build_nvrs(self, errata_id, only_modules=False):
+        """ Get module build NVRs which are affected by the CVEs in errata
+
+        :param errata_id: Errata advisory ID, e.g. 25713.
+        :type errata_id: str or int
+        :param only_modules: only return module builds NVR if True.
+        :type only_modules: bool
+        :return: a list of strings each of them is a build NVR
+        :rtype: list
```

Comment:
Does it really need to return a list? Please consider if `set` wouldn't be enough.

Reply:
Thanks @mvalik , please check it in above reply.

Reply:
If it's an overwritten method of the base class then it's OK

Reply:
For an unknown reason I don't see the 'Resolve' button, please consider this particular thread as resolved.